src/ScheduleFunctions.cpp

```diff
@@ -534,6 +540,26 @@ Stmt build_produce(Function f) {
             }
         }
 
+        Stmt annotate;
+        if (target.has_feature(Target::MSAN)) {
+            // Mark the buffers as initialized before calling out.
```

Comment:
Didn't do this since I don't think we want to annotate the buffer_t for ImageParam either.